src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -1013,10 +1013,72 @@ zocl_load_aie_only_pdi(struct drm_zocl_dev *zdev, struct axlf *axlf,
 }
 
 int
-zocl_pl_only_reset(struct drm_zocl_dev *zdev, const char *buf, size_t count)
+zocl_reset(struct drm_zocl_dev *zdev, const char *buf, size_t count)
```

Comment:
Why you decide to add zocl_reset() function in zocl_xclbin.c? Is this file is used for handling xclbin related work?

Reply:
> Why you decide to add zocl_reset() function in zocl_xclbin.c? This file is used for xclbin related work. This function is handling clients and slots but not xclbin related. Any reason?

As in this function we are mostly cleanup xclbin related data structure I have added this in this file.  Please feel free to suggest any better place to add this function.

Reply:
Since we have client and slots in this function, can we move it into zocl_kds.c?
For xclbin related cleanup, you can create a function in zocl_xclbin.c, like zocl_xclbin_cleanup().